.github/workflows/main.yml

```diff
@@ -29,95 +31,71 @@ jobs:
 
-      - name: Deploy a sample application for chaos injection
+      # Starting the React-Server for E2E tests.
+      - name: Starting the server
+        if: startsWith(github.event.comment.body, '/run-e2e') || startsWith(github.event.comment.body, '/run-all')
         run: |
-          kubectl apply -f https://raw.githubusercontent.com/mayadata-io/chaos-ci-lib/master/app/nginx.yml
-          sleep 30
-          
-      - name: Setting up kubeconfig ENV for Github Chaos Action
-        run: echo ::set-env name=KUBE_CONFIG_DATA::$(base64 -w 0 ~/.kube/config)
-
-      - name: Running Litmus pod delete chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-pod-delete') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: pod-delete
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true
-
-      - name: Running container kill chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-container-kill') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: container-kill
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          CONTAINER_KILL_HELPER_IMAGE: litmuschaos/container-kill-helper
-          CONTAINER_KILL_HELPER_TAG: latest
-          LIB: containerd
-          LITMUS_CLEANUP: true
-
-      - name: Running node-cpu-hog chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-node-cpu-hog') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: node-cpu-hog
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true
-
-      - name: Running node-memory-hog chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-node-memory-hog') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: node-memory-hog
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true      
-                      
+          CHOKIDAR_USEPOLLING=1 npm start &
+        working-directory: litmus-portal/frontend
+
+      - name: Installing Cypress and running tests for login page
+        if: startsWith(github.event.comment.body, '/run-e2e-login')
```

Comment:
if condition says it'll run for `run-e2e-login`  but what about `run-e2e-all` should we add this condition also?

Reply:
Actually, in cypress, I can run all test scripts in one go, I don't have to start cypress for each test again and again, so that's why what I did is separated the run-e2e-feature cases and run-e2e-all in one case.

- run-e2e-all will start cypress only once and run all the tests.
- run-e2e-feature will start cypress for each test script, which will take time in between the tests.[ that's why they are not included in run-all, only e2e-all is included in that case. ]